docs/guides/guides.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Guides
+description: The Amplify Command Line Interface (CLI) is a unified toolchain to create, integrate, and manage the AWS cloud services for your app. The CLI is category-based with best practices built in. 
+disableTOC: true
+filterKey: platform
+---
+
+Amplify guides are meant to give you a more in depth understanding of how to use the Amplify CLI, libraries, and hosting to build out common functionality, end to end solutions, and frequently asked for workflows.
```

Comment:
"and hosting" should we say "Console" or "hosting capability". Just "hosting" makes it seems like we have a product named Amplify hosting.